pyfolio/pos.py

```diff
@@ -126,12 +136,18 @@ def extract_pos(positions, cash):
 
     positions = positions.copy()
     positions['values'] = positions.amount * positions.last_sale_price
+
     cash.name = 'cash'
 
     values = positions.reset_index().pivot_table(index='index',
                                                  columns='sid',
                                                  values='values')
 
+    if ZIPLINE:
+        for asset in values.columns:
+            if type(asset) in [Equity, Future]:
```

Comment:
Since `price_multiplier` exists for all `Asset`s, what's your thought on the subtype check here? (And is it intentionally excluding subclasses of these subclasses - versus`if isinstance(asset, (Equity, Future))`?)

Reply:
Hey @richafrank, You make a good point here. Initially I was only checking for `Future` as that was the specific case I was trying to solve for. Then I checked the `Equity` class and saw that it also had a `price_multiplier` so I simply added that. Would you recommend changing this to checking `issubclass(asset, Asset)`? To think of it, `pf.utils.extract_rets_pos_txn_from_zipline(results)` is predicated on one passing a results object from a zipline sim. As such, would it be appropriate to just assume therefore that these columns are `Asset`s and not even check?